fern/apis/testnet_rest/openapi/openapi.json

```diff
@@ -5455,7 +5455,7 @@
           "description": "Minimum share percentage (0-100) the vault owner must maintain on the vault",
           "type": "string",
           "example": "5"
-        },
+                },
```

Comment:
Not intentional?

Reply:
yeah copy/paste gone wrong :disappointed:

Reply:
fixed